pages/api/courses/[course_id].js

```diff
@@ -2,22 +2,56 @@
 // DELETE: /pages/api/courses/[course_id].js
 // PUT: /pages/api/courses/[course_id].js
 
-export default function getRemoveAndUpdateCourse(req, res) {
+import { db } from '../../../utils/api/firebase.config';
+import {
+  collection,
+  doc,
+  getDoc,
+  getDocs,
+  query,
+  where,
+  deleteDoc,
+  updateDoc,
+} from "firebase/firestore";
+
+export default async function getRemoveAndUpdateCourse(req, res) {
   const {
     query: { course_id },
     method
   } = req;
 
   switch (method) {
     case 'GET':
-      // open to add query
+      /* getCoursesById */
+      try {
+        // id field is Number value
+        const q = query(collection(db, 'courses'), where('id', '==', Number(course_id)));
```

Comment:
Single letter variables should only be used in a few select instances, such as for loops and synthetic events. Rename 'q' to more descriptive name.

Reply:
@cmjosephs - this was what the Firestore official docs used for their exact syntax. We can change if needed but also it might be helpful to be able to reference official docs.